index.js

```diff
@@ -55,7 +80,12 @@
 // ***** Question 5 *****
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 5", "color: red")
+
 // const student1 = { name: "Duane", grade: 88 }
+// function updateGrade(student, grade) {
+//   student.grade = grade
+// }
```

Comment:
nice job using JS's dot notation
remember that the return value of this function as written will be undefined. This may not be a problem if we don't need the return value. Just pointing it out.